res/TensorFlowLiteRecipes/Net_Decompose_HardSwish_000/test.recipe

```diff
@@ -0,0 +1,17 @@
+operand {
```

Comment:
`Net_` prefix recipes are used for a (small) network models, say, with two or more operations.
IMHO, we can just use `HardSwish_001` for decomposition testing.

Reply:
Thanks for pointing out the meaning of `Net_` prefix.
I will change the name =)

Reply:
Applied in 58c7640